tests/stress/km/stress_tests_km.cpp

```diff
@@ -1550,4 +1751,4 @@ TEST_CASE("bindmonitor_tail_call_invoke_program_test", "[native_mt_stress_test]"
     _print_test_control_info(local_test_control_info);
     _mt_bindmonitor_tail_call_invoke_program_test(local_test_control_info);
 }
-#endif
+#endif // ENABLE_TAIL_CALL_STRESS_TEST
```

Comment:
NOTE: This PR does NOT resolve the issues with this testcase. It still appears to be broken/failing.